pom.xml

```diff
@@ -223,8 +227,10 @@
                         <headerLocation>gradoop/LICENSE.txt</headerLocation>
                         <includeResources>false</includeResources>
                         <includeTestResources>false</includeTestResources>
-                        <suppressionsLocation>gradoop/checkstyle-suppressions.xml</suppressionsLocation>
-                        <suppressionsFileExpression>checkstyle.suppressions.file</suppressionsFileExpression>
+                        <suppressionsLocation>gradoop/checkstyle-suppressions.xml
+                        </suppressionsLocation>
+                        <suppressionsFileExpression>checkstyle.suppressions.file
```

Comment:
If you make a line break, move content to a new line between opening and closing tag